services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/DirectionsJsonObject.java

```diff
@@ -33,6 +34,25 @@ public String toJson() {
     return gson.create().toJson(this);
   }
 
+  /**
+   * Access JSON property that wasn't recognised during JSON serialization.
+   * This may be useful to access experimental properties.
+   * @param propertyName name of a json property
+   * @return value of a requested property or null if the requested property doesn't exist.
+   */
+  @Nullable
+  public JsonElement getUnrecognisedProperty(String propertyName) {
+    JsonElement result = null;
+    Map<String, SerializableJsonElement> recognisedProperties = unrecognised();
+    if (recognisedProperties != null) {
+      SerializableJsonElement property = recognisedProperties.get(propertyName);
+      if (property != null) {
+        result = property.getElement();
+      }
+    }
+    return result;
+  }
```

Comment:
```suggestion
  /**
   * Access JSON property that wasn't recognized during JSON deserialization.
   * This may be useful to access experimental properties.
   * @param propertyName name of a json property
   * @return value of a requested property or null if the requested property doesn't exist.
   */
  @Nullable
  public JsonElement getUnrecognizedProperty(String propertyName) {
    JsonElement result = null;
    Map<String, SerializableJsonElement> recognizedProperties = unrecognized();
    if (recognizedProperties != null) {
      SerializableJsonElement property = recognizedProperties.get(propertyName);
      if (property != null) {
        result = property.getElement();
      }
    }
    return result;
  }
```
:upside_down_face:

Reply:
oh, again 🤦